### PR TITLE
Feature/error handling

### PR DESCRIPTION
**Pull request by SarthakSingh31**

### CODE REVIEW

File: app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt [hunk 1]
```diff
@@ -197,4 +191,28 @@ class CommonWebViewFragment: Fragment() {
...
+                ErrorActivity.ErrorHandlerEnum.RELOAD_PAGE,
+                it
+            )
+        }
+    }
+
+    private fun raiseUnableToConnect() {
+        activity?.let {
+            ErrorActivity.showError(
+                "Unable to Load Page",
```
Comment (DDaiJ) [inline app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt]: We have these warnings in strings.xml

File: app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt [hunk 2]
```diff
@@ -197,4 +191,28 @@ class CommonWebViewFragment: Fragment() {
             }
         super.onResume()
     }
-}
\ No newline at end of file
+
+    private fun raiseNoInternet() {
+        activity?.let {
+            ErrorActivity.showError(
+                "No Internet Connection",
```
Comment (DDaiJ) [inline app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt]: Should go into strings.xml.

File: app/src/main/java/edu/uw/myuw_android/UserInfoStore.kt [hunk 3]
```diff
@@ -118,28 +112,53 @@ object UserInfoStore {
...
+            }
+            Log.d("updateAffiliations - decodedRespose: ", responseJson)
+            Log.d("updateAffiliations - affiliations: ", affiliations.toString())
+        }
+        catch (e: Exception) {
+            Log.e("updateAffiliations - http error", e.toString())
+            InternetCheck {
+                if (it) {
+                    ErrorActivity.showError(
+                        "Unable to Load Page",
```
Comment (DDaiJ) [inline app/src/main/java/edu/uw/myuw_android/UserInfoStore.kt]: strings.xml and same for below

**Review by DDaiJ — COMMENTED** [review]
Do we need to add error handlers for search page? Because right now it will say webpage not available and gives ERR_ADDRESS_UNREACHABLE when there's no internet connection

File: app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt (hunk 1 above)
Comment (SarthakSingh31) [reply app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt]: done

File: app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt (hunk 2 above)
Comment (SarthakSingh31) [reply app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt]: done

File: app/src/main/java/edu/uw/myuw_android/UserInfoStore.kt (hunk 3 above)
Comment (SarthakSingh31) [reply app/src/main/java/edu/uw/myuw_android/UserInfoStore.kt]: done

**Review by DDaiJ — DISMISSED** [review]
found two bugs that need high priority fix

**Review by DDaiJ — APPROVED** [review]

**Pull request merged**